pex/interpreter.py

```diff
@@ -267,6 +267,10 @@ class PythonInterpreter(object):
 
     re.compile(r'python[23]$'),
     re.compile(r'python[23].[0-9]$'),
+
+    # PEP-3149: Python implementations MAY include additional flags in the file name tag
+    re.compile(r'python[23].[0-9][a-z]$'),
```

Comment:
You've found cases of this in the wild and that may be more important to mention since, IIUC, that pep refers to so names and not python interpreter names.